api/product/views.py

```diff
@@ -109,7 +109,7 @@ def get_class_tree_by_group(self, request):
             # TODO retornar execpt que o group e obrigatorio
             return Response({
                 'success': False,
-                'msg': 'Necessário passar o parametro group.'
+                'msg': 'NecessÃ¡rio passar o parametro group.'
             })
 
```

Comment:
Nessa Classe, nesse trecho 
https://github.com/linea-it/dri/blob/develop/api/product/views.py#L151
vc deve fazer um if assim como esse https://github.com/linea-it/dri/blob/develop/api/product/views.py#L145
usando o request.user.pk vc vai Fazer Bookmark.objects.get(produto=iddoproduto, owner=request.user.pk)
o get tem que ficar entre try execpt se der certo vc adiciona no dict catalog  is_starred = true e o iconCls vc coloca a classe css do icone de estrela, caso o get de except vc coloca o starred como false e o iconCls como no-icon
acerta os nomes de acordo com o que vc esta usando no codigo agora.